kokoro/linux/dockerfile/test/csharp/Dockerfile

```diff
@@ -22,14 +22,18 @@ RUN apt-get update && apt-get install -y \
   wget \
   && apt-get clean
 
+# Update ca-certificates to fix known buster + .NET 5 issue
+# https://github.com/NuGet/Announcements/issues/49
+RUN apt-get update && apt-get install -y ca-certificates && apt-get clean
+
 # dotnet SDK prerequisites
-RUN apt-get update && apt-get install -y libunwind8 libicu57 && apt-get clean
```

Comment:
libicu57 is not longer available in buster? .NET SDK installs so not needed? Or will the problem show up when building/running .NET apps? 🤷

Reply:
added libicu63 instead